guides/README.md

```diff
@@ -71,12 +73,14 @@ To generate the stylesheet:
 	git checkout foreman-css
 	compass compile
 
-## Contributing
+## Contribution guidelines
+
+Please read these guidelines before opening a Pull Request. For more information, see [Guidelines for Red Hat Documentation](https://redhat-documentation.github.io/).
 
-Please read [Guidelines for Red Hat Documentation](https://redhat-documentation.github.io/) before opening a Pull Request.
-Additional rules apply:
+### Variables
 
-Never write "Foreman" or "Satellite" words directly but use the following variables:
+The content in this repository is shared between the community Foreman docs and the Red Hat Satellite docs.
```

Comment:
ATIX will also be adding content and are using it, so perhaps leave it a bit open to Foreman, Satellite, and anyone else who would like to use and contribute to these docs.

Reply:
Something like:

```suggestion
The content in this repository is not just for the upstream Foreman community but is also used in branded downstream products such as Red Hat Satellite and ATIX Orcharino.
```

@maximiliankolb I'd appreciate a look at the ATIX note.

Reply:
I'd suggest `orcharhino by ATIX`.